### ISSUE TITLE
Some annotations are not accurate

### ISSUE THREAD

**Issue by zzpustc** [SilvioGiancola/TrackingNet-devkit#5]
Hello, I find there always exists some frames with wrong annotations, are these frames annotated by SOTA trackers? I think these annotations would hurt the performance of deep learning based trackers.

**Comment by zzpustc** [comment]
@SilvioGiancola

**Comment by SilvioGiancola** [comment]
Dear @zzpustc
Could you please share with us the IDs of the sequences you are referring to, so we investigate your concern? I believe you are referring to the training set since you are mentioning our interpolation method.

**Comment by zzpustc** [comment]
@SilvioGiancola For example, in the TRAIN_0/frames/A63BoLTUNAM_0,a1qoB1eERn0_0, the annotations are absolutely wrong, there also exists some annotations are not accurate, such as TRAIN_0/frames/A3E72P24pf8_0.

**Comment by zzpustc** [comment]
@SilvioGiancola  Hope to receive your response, since I have use trackingnet as my training dataset, but it does not converge.

**Comment by SilvioGiancola** [comment]
@zzpustc , we are currently investigating the sequences you were mentioning. As mentioned in our paper, we are using an interpolation method between manually annotated frames. Such interpolation, even though not perfect, has been shown to provide enough information to improve state-of-the-art trackers.
I don't believe a weak annotation will impede your algorithm to converge. Regardless of the data, your model and your loss should be able to converge. If you believe those sequences are not beneficial for your algorithm, feel free not to use them. Have you already tried to overfit a model with a limited portion of the data?

**Comment by zzpustc** [comment]
@SilvioGiancola Sorry , it's my mistake, there are some bugs in my code, thank you for patient reply.

**Comment by SilvioGiancola** [comment]
@zzpustc I am glad you found your bug on your code, do not hesitate to contact us again for further inquiries. 
Best
